CONTRIBUTING.md

```diff
@@ -17,6 +17,14 @@ always take into consideration how these new features affect users and whether
 or not adding a new section or document to the documentation (found under the
 `doc` folder) would be useful.
 
+###Repository branch structure
```

Comment:
I just noticed that the `###` is not being rendered as a title, see [here](https://github.com/josenavas/QiiTa/blob/73dca6fc78ca29a70725667ca916f7464618acd0/CONTRIBUTING.md). Seems to be a problem with a missing space between `###` and the first letter in the title.